.env.example

```diff
@@ -8,6 +8,7 @@ VITE_OISY_URL=https://oisy.com
 VITE_ERC20_ICP_SYMBOL=ckICP
 VITE_ERC20_ICP_REPO_URL=https://github.com/dfinity/erc20-icp
 VITE_COINGECKO_API_URL=https://api.coingecko.com/api/v3
+VITE_BLOCKCHAIN_API_URL=https://blockchain.info
```

Comment:
That's correct? Their api endpoint is the same URL as their website?

Reply:
Surprisingly, yes, it's the same URL.

Reply:
Weird there they say it's `api.blockchain.com` 🤷‍♂️
https://api.blockchain.com/v3/#authentication

Maybe I'm looking at the wrong documentation. Thanks for the feedback.